### PR TITLE
add neutropenia grading function

### PR DESCRIPTION
**Pull request by jasmineirx**
This function applies the NCI's Common Toxicity Criteria to a vector of ANCs to return numeric neutropenia grade.

### CODE REVIEW

File: R/calc_neutropenia_grade.R [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#' Calculate neutropenia grade from ANC
+#'
+#' Assigns neutropenia grade based on the National Cancer Institute system. Note
+#' that while this system assigns a grade of 1 to an ANC between 1500-2000, the
+#' term neutropenia is usually reserved for a grade of 2 or higher (an ANC of
+#' <1500)
+#'
+#' @param anc absolute neutrophil count (ANC), in number per microliter
```
Comment (roninsightrx) [inline R/calc_neutropenia_grade.R:8]: is this really the most common unit? I'm more used to seeing ANC in terms of 1.e9 counts/L, so grading of 0.5, 1, 1.5, 2.

**Review by roninsightrx — CHANGES_REQUESTED** [review]
- one question about unit
- can we add to reference in pkgdown?

File: R/calc_neutropenia_grade.R (hunk 1 above)
Comment (roninsightrx) [reply R/calc_neutropenia_grade.R:8]: it's fine, see now that CTC-AE v5 lists both units. Not a big deal for user to convert between values.

Comment (jasmineirx) [reply R/calc_neutropenia_grade.R:8]: Perhaps one other design question is whetehr we prefer to return 0 or NA. For AKI we return NA, and I don't think "grade 0" really has a meaning, but 0 is a bit easier to work with.

Comment (roninsightrx) [reply R/calc_neutropenia_grade.R:8]: yes thought about that but was ok with the NA. Technically grade 0 doesn't exist in CTC-AE, so NA is more correct IMO.

**Review by roninsightrx — APPROVED** [review]
lgtm

**Pull request merged**